ReactWindows/ReactNative/Views/DatePicker/ReactCalendarDatePickerShadowNode.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Facebook.Yoga;
+using ReactNative.UIManager;
+
+namespace ReactNative.Views.DatePicker
+{
+    /// <summary>
+    /// The shadow node implementation for <see cref="ReactNative.Views.DatePicker.ReactCalendarDatePickerManager" /> views.
+    /// </summary>
+    public class ReactCalendarDatePickerShadowNode : LayoutShadowNode
+    {
+        /// <summary>
+        /// Instantiates the <see cref="ReactDatePickerShadowNode"/>.
+        /// </summary>
+        public ReactCalendarDatePickerShadowNode()
+        {
+            MeasureFunction = MeasurePicker;
+        }
+
+        private static YogaSize MeasurePicker(YogaNode node, float width, YogaMeasureMode widthMode, float height, YogaMeasureMode heightMode)
+        {
+            return MeasureOutput.Make(width, 40f);
+        }
```

Comment:
I could not find any case in which the height might be undefined. If the height is defined in JavaScript, the component occupies the area specified by the value, but the picker itself keeps its size.